src/Modules/UngradedModules.jl

```diff
@@ -1443,15 +1490,9 @@ end
 @doc Markdown.doc"""
     issubset(M::SubQuo{T}, N::SubQuo{T}) where T
 
-Return `true` if `M` is contained in `N`, and `false` otherwise.
-
-The basic requirements for `true` are that `ambient_free_module(M) === ambient_free_module(N)`,
-and that the relations of `M` and `N` generate the same submodule of this free module. 
-If, say, 
-
-$M = (\text{im } a_M + \text{im } b_M)/\text{im } b_M \text{ and } N = (\text{im } a_N + \text{im } b_N)/\text{im } b_N, \text{ with } \text{im } b_M = \text{im } b_N,$ 
-
-it is moreover checked whether $\text{im } a_M + \text{im } b_M$ is contained in $\text{im } a_N + \text{im } b_N$.
+Given subquotients `M` and `N` such that `ambient_module(M) == ambient_module(N)`,
```

Comment:
And what happens if the ambient modules are not equal?

Reply:
@fingolfin I will add a sentence starting with Otherwise as soon as this is merged.